src/main/java/com/mabl/integration/jenkins/validation/MablRestApiClientRetryHandler.java

```diff
@@ -0,0 +1,37 @@
+package com.mabl.integration.jenkins.validation;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.ServiceUnavailableRetryStrategy;
+import org.apache.http.impl.client.DefaultServiceUnavailableRetryStrategy;
+import org.apache.http.protocol.HttpContext;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.apache.commons.httpclient.HttpStatus.SC_NOT_IMPLEMENTED; // 501
+import static org.apache.commons.httpclient.HttpStatus.SC_BAD_GATEWAY; // 502
+
+public class MablRestApiClientRetryHandler implements ServiceUnavailableRetryStrategy {
+    private final ArrayList<Integer> retryStatusCodes = new ArrayList<Integer>(Arrays.asList(
+            SC_NOT_IMPLEMENTED,
+            SC_BAD_GATEWAY
+    ));
+    private static final int maxRetries = 5;
+    private static final long retryInterval = 1000L;
```

Comment:
I assume this is milliseconds. We've been trying to use units where applicable in all variable names.